cmd/server/assets/realmadmin/show.html

```diff
@@ -23,7 +23,16 @@ <h1>Realm stats</h1>
     </p>
 
     <div class="card mb-3 shadow-sm">
-      <div class="card-header">Statistics</div>
+      <div class="card-header">Statistics
+				<div style="padding-left: 50px;">
+				  Realm Stats
+				  (<a href="/realm/stats.csv">CSV</a>/<a href="/realm/stats.json">JSON</a>)
+				</div>
+				<div style="padding-left: 50px;">
+				  Realm Per-User Stats
```

Comment:
```suggestion
				  Realm per-user stats
```

We've only been using caps for the first word for titles